sisl/grid.py

```diff
@@ -195,6 +218,76 @@ def interp(self, shape, method='linear', **kwargs):
 
         return grid
 
+    def smooth(self, method='gaussian', r=None, mode='wrap', **kwargs):
+        '''
+        Make a smoother grid by applying a filter.
+
+        Parameters
+        -----------
+        method: {'gaussian', 'uniform'}, optional
+            the type of filter to apply to smoothen the grid.
+        r: float or array-like of float, optional
+            the radius of the filter in Angstrom for each axis.
+            If the method is 'gaussian', this is the standard deviation!
+            
+            If a single float is provided, then the same distance will be used for all axes.
+            
+            The defaults are 0.6 Ang for a gaussian filter and 1.5 Ang for a uniform filter.
+        mode: {'wrap', 'mirror', 'constant', 'reflect', 'nearest'}, optional
+            determines how to compute the borders of the grid.
+            The default is wrap, which accounts for periodic conditions. 
+            
+            See https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.gaussian_filter.html#scipy.ndimage.gaussian_filter
+        '''
+        
+        if r is None:
+            r = 0.6 if method == 'gaussian' else 1.5
```

Comment:
write this out ;)

Reply:
Do you mean to not have a default for `r` or that I should use more than one line for the if/else? 😅  By the way I just realized that I am calculating the kernel size wrong for the uniform case, if `r` is the radius, the size should be doubled (default radius halved).

Reply:
Hehe, two lines.  
Or you could do:
``python
if r is None:
    r = {"gaussian": 0.6}.get(method, 1.5)
```
in which case it becomes easy to extend defaults?